app/src/main/main.js

```diff
@@ -293,15 +293,22 @@ ipcMain.on('move-cropper-window', (event, data) => {
   }
 
   const position = cropperWindow.getPosition();
-  if (data.direction === 'left') {
-    position[0] -= data.amount;
-  } else if (data.direction === 'up') {
-    position[1] -= data.amount;
-  } else if (data.direction === 'right') {
-    position[0] += data.amount;
-  } else if (data.direction === 'down') {
-    position[1] += data.amount;
+  const amount = data.amount;
+  
+  switch(data.direction){
```

Comment:
Please add a space between the switch and the parentheses and the parentheses and the bracket.